Helpers/TaskHelpers.cs

```diff
@@ -94,15 +131,17 @@ private static void handleAggregateWithNotify(string failureMessage, string canc
                 if (x is APIException) // This we know how to handle.
                 {
                     if (!String.IsNullOrEmpty(failureMessage))
-                        TrayIcon.NotifyError(
-$@"{failureMessage}
-{x.Message}");
+                        TrayIcon.NotifyError($@"{failureMessage} {x.Message}");
 
                     return true;
                 }
 
+                if (retryable)
+                    return false;

```

Comment:
Explain what this does. It does not mark the timeout contained in the aggregate as handled.

Reply:
Naming is confusing, as mentioned on the call we had. It does not convey the actual intent.